spec/lib/miq_automation_engine/service_methods/miq_ae_service_tenant_spec.rb

```diff
@@ -28,5 +29,12 @@ module MiqAeServiceTenantSpec
     it "#description" do
       expect(service_tenant.description).to eq('Krueger')
     end
+
+    it "#tenant_quotas" do
+      cpu_quota = TenantQuota.create(:name => "cpu_allocated", :unit => "int", :value => 2, :tenant_id => tenant.id)
+      storage_quota = TenantQuota.create(:name => "storage_allocated", :unit => "GB", :value => 160, :tenant_id => tenant.id)
```

Comment:
@mkanoor These should be using FactoryGirl as well.  You will need to create the factory in /spec/factories for tenant_quota.